integration_tests/tests/integrity/vertical_integrity_daily_shop_fulfillment.sql

```diff
@@ -0,0 +1,80 @@
+{{ config(
+    tags="fivetran_validations",
+    enabled=var('fivetran_validation_tests_enabled', false)
+) }}
+
+with source as (
+
+    select *
+    from {{ ref('stg_shopify__fulfillment_event') }}
+),
+
+source_metrics as (
+
+    select 
+        source_relation,
+        cast({{ dbt.date_trunc('day','happened_at') }} as date) as date_day,
+        shop_id,
+        count(distinct case when status = 'delayed' then fulfillment_id end) as count_fulfillment_delayed,
+        count(distinct case when status = 'in_transit' then fulfillment_id end) as count_fulfillment_in_transit,
+        count(distinct case when status = 'confirmed' then fulfillment_id end) as count_fulfillment_confirmed,
+        count(distinct case when status = 'delivered' then fulfillment_id end) as count_fulfillment_delivered
+
+    from source
+    where happened_at > '2023-01-01' and happened_at < '2024-01-01'
```

Comment:
What is the reason for this filter? My only concern with this filter is if you run the integrity validation on a data set that doesn't have data for this period then it won't validate the results effectively. Is there a concern for removing this filter and running the validation for all records?

Reply:
Same comment for the other integrity tests

Reply:
ah this was just me avoiding live-data messing things up, will adjust to be wider